src/exercises/adapt-c/Makefile

```diff
@@ -0,0 +1,14 @@
+CC=clang
+
+ARCH!=uname -n
+.if ${ARCH} == "cheribsd-morello-purecap"
```

Comment:
Uh, what, you're using the hostname? What's wrong with uname -p to get MACHINE_ARCH or uname -m to get MACHINE (since you don't actually care about aarch64 vs aarch64c vs riscv64 vs riscv64c, just arm64 vs riscv)? Though really you shouldn't need this, just use cc, then your CFLAGS can be machine-independent, and just hybrid vs purecap needs to be machine-dependent.